### DISCUSSION TITLE
How to call an identity server protected API resource from within ELSA workflow's SendHttpRequest activity using bearer token

### DISCUSSION THREAD

**Discussion by zyphertechnologies** [elsa-workflows/elsa-core#2463] [Q&A]
My detailed question is posted [here](https://stackoverflow.com/questions/69893460/how-to-call-an-identity-server-protected-api-resource-from-within-elsa-workflow)

**Comment by sfmskywalker** [comment]
[Answered](https://stackoverflow.com/a/69903667/690374) by [AndrewSilver](https://stackoverflow.com/users/1752270/andrewsilver)
